pkg/ccr/record/recover_info.go

```diff
@@ -0,0 +1,42 @@
+package record
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/selectdb/ccr_syncer/pkg/xerror"
+)
+
+type RecoverInfo struct {
+	DbId             int64  `json:"dbId"`
+	NewDbName        string `json:"newDbName"`
+	TableId          int64  `json:"tableId"`
+	TableName        string `json:"tableName"`
+	NewTableName     string `json:"newTableName"`
+	PartitionId      int64  `json:"partitionId"`
+	PartitionName    string `json:"partitionName"`
+	NewPartitionName string `json:"newPartitionName"`
+}
+
+func NewRecoverInfoFromJson(data string) (*RecoverInfo, error) {
+	var recoverInfo RecoverInfo
+	err := json.Unmarshal([]byte(data), &recoverInfo)
+	if err != nil {
+		return nil, xerror.Wrap(err, xerror.Normal, "unmarshal create table error")
+	}
+
+	if recoverInfo.TableId == 0 {
+		return nil, xerror.Errorf(xerror.Normal, "table id not found")
+	}
+
+	/* need check for db level not supported.
+	 */
```

Comment:
Please add the above check, and also check if the table name and partition name exist.